gtsam/inference/BayesTree.h

```diff
@@ -269,6 +268,12 @@ namespace gtsam {
 
     /// @}
 
+#ifdef GTSAM_ALLOW_DEPRECATED_SINCE_V42
+  public:
+    typedef GTSAM_DEPRECATED Clique Node; ///< Synonym for Clique (TODO: remove)
+    typedef GTSAM_DEPRECATED sharedClique sharedNode; ///< Synonym for sharedClique (TODO: remove)
```

Comment:
This change is the breaker. We should undo it since it is a much bigger effort which is also tangential to this PR.

Reply:
Indeed, it seems unrelated and should be it’s own PR. If it is in fact appropriate, because some of the parallel processing stuff might need this.